mlem/cli/deployment.py

```diff
@@ -64,21 +87,75 @@ def deploy_run(
     """
     from mlem.api.commands import deploy
 
-    conf = conf or []
-    env_conf = [c[len("env.") :] for c in conf if c.startswith("env.")]
-    conf = [c for c in conf if not c.startswith("env.")]
     deploy(
-        path,
-        model,
-        env,
-        project,
+        load,
+        load_meta(
+            model, project=model_project, rev=model_rev, force_type=MlemModel
+        ),
+        project=project,
+        rev=rev,
         external=external,
         index=index,
-        env_kwargs=parse_string_conf(env_conf),
-        **parse_string_conf(conf or []),
     )
 
 
+@for_each_impl(MlemDeployment)
+def create_deploy_run_command(type_name):
+    @mlem_command(
+        type_name,
+        section="deployments",
+        parent=deploy_run,
+        dynamic_metavar="__kwargs__",
+        dynamic_options_generator=add_env_params_deployment(
+            type_name, MlemDeployment
+        ),
+        hidden=type_name.startswith("_"),
+        lazy_help=lazy_class_docstring(MlemDeployment.object_type, type_name),
+        no_pass_from_parent=["file_conf"],
+    )
+    def deploy_run_command(
+        path: str = Argument(
+            ..., help="Where to save the object (.mlem file)"
+        ),
+        model: str = make_not_required(option_model),
+        model_project: Optional[str] = option_model_project,
+        model_rev: Optional[str] = option_model_rev,
+        project: Optional[str] = option_project,
+        external: bool = option_external,
+        index: bool = option_index,
+        file_conf: List[str] = option_file_conf("deployment"),
+        **__kwargs__,
+    ):
+        from mlem.api.commands import deploy
+
+        try:
+            meta = load_meta(path, project=project, force_type=MlemDeployment)
+            raise DeploymentError(
+                f"Deployment meta already exists at {meta.loc}. Please use mlem deployment run --load <path> ..."
```

Comment:
Should we quote the commands in errors? like 
```suggestion
                f"Deployment meta already exists at {meta.loc}. Please use `mlem deployment run --load <path> ...`"
```